packages/cli-e2e/entrypoints/entry.js

```diff
@@ -57,7 +57,11 @@ try {
   execSync(
     `docker run --name=${DOCKER_CONTAINER_NAME} -v "${repoHostPath}:${repoDockerPath}" -p "9229:9229" -${
       process.argv[2] === '--bash' ? 'it' : 'i'
-    } --cap-add=SYS_ADMIN ${DOCKER_IMAGE_NAME} ${dockerEntryPoint}`,
+    } --cap-add=SYS_ADMIN --env PLATFORM_USER_NAME=${
```

Comment:
TODO: pass env variable in .env file

Reply:
it's just to prevent having a really long command. But this is not important. I can put this in the backlog.

Reply:
I think you could include https://www.npmjs.com/package/dotenv and that'd be way easier and cleaner than now.

How's it's done is really too hackish imho.